test/wallet-http-test.js

```diff
@@ -839,6 +845,75 @@ describe('Wallet HTTP', function() {
     matchTxId(auction.reveals, state.reveals[1].hash);
   });
 
+  it('should create a bid and a reveal (reveal in advance)', async () => {
+    const balanceBeforeTest = await wallet.getBalance(null);
```

Comment:
`getBalance()` can be left empty I think, `null` is probably fine but redundant.

Reply:
Will try it